api/client/builder/types.go

```diff
@@ -1307,6 +1275,208 @@ func (p *ExecutionPayloadDeneb) ToProto() (*v1.ExecutionPayloadDeneb, error) {
 	}, nil
 }
 
+// ExecHeaderResponseElectra is the header response for builder API /eth/v1/builder/header/{slot}/{parent_hash}/{pubkey}.
+type ExecHeaderResponseElectra struct {
```

Comment:
Why cant the testing middleware uses this struct here? 
https://github.com/prysmaticlabs/prysm/blob/5bbbad65bc2287c50a8a17e4dd8410ad5ea75427/testing/middleware/builder/builder.go#L109

Reply:
good question i'm not sure i should fix the old ones too